modules/administration/pages/users.adoc

```diff
@@ -30,9 +34,17 @@ Deactivated users cannot log in to the {productname} {webui} or schedule any act
 Actions scheduled by a user prior to their deactivation remain in the action queue.
 Deactivated users can be reactivated by {productname} administrators.
 
+
+
+== Administrator Roles
+
 Users can hold multiple administrator roles, and there can  be more than one user holding any administrator role at any time.
 There must always be at least one active {productname} Administrator.
 
+To change a user's administrator roles, except for the {productname} Administrator role, navigate to menu:Users[User List > All], select the user to change, and check or uncheck the administrator roles as required.
+
+To change a user's {productname} or Organization Administrator role, navigate to menu:Admin[Users] and check or uncheck [guimenu]``{productname} Admin?`` or [guimenu]``{Org Admin}?`` as required.
```

Comment:
Org Admin can be modified at the other place AFAIK. Only the SUSE Manager Administrator is special

Reply:
So, by my testing, you can adjust the Org Admin role in both places, but can only adjust the SUMA Admin role in Admin (not in Users). So on L44, I've said that the only role you *can't* adjust in Users is the SUMA Admin role, and on L46, I've said that you can adjust *both* in Admin. Have I misunderstood?

Reply:
oh! lol. I'll fix it :)